manifests/grafana-dashboard-nginx.yaml

```diff
@@ -19,6 +19,12 @@ data:
             "hide": true,
```

Comment:
Can you revert this file to the original one?

The dashboard works for me, and the change in the file is difficult to follow.

Reply:
Done